security/browser-profiles/android_webview_83.json

```diff
@@ -0,0 +1,2533 @@
+                        },
+                        "preventSilentAccess": {
+                            "type": "function",
+                            "value": "function () { [native code] }"
+                        },
+                        "store": {
+                            "type": "function",
+                            "value": "function () { [native code] }"
+                        }
+                    }
+                },
+                "deviceMemory": {
+                    "type": "number"
+                },
+                "doNotTrack": {
+                    "type": "object"
+                },
+                "geolocation": {
+                    "type": "object",
+                    "properties": {
+                        "clearWatch": {
+                            "type": "function",
+                            "value": "function clearWatch() { [native code] }"
+                        },
+                        "getCurrentPosition": {
+                            "type": "function",
+                            "value": "function getCurrentPosition() { [native code] }"
+                        },
+                        "watchPosition": {
+                            "type": "function",
+                            "value": "function watchPosition() { [native code] }"
+                        }
+                    }
+                },
+                "getBattery": {
+                    "type": "function",
+                    "value": "function getBattery() { [native code] }"
+                },
+                "getGamepads": {
+                    "type": "function",
+                    "value": "function getGamepads() { [native code] }"
+                },
+                "getUserMedia": {
+                    "type": "function",
+                    "value": "function () { [native code] }"
+                },
+                "hardwareConcurrency": {
+                    "type": "number"
+                },
+                "javaEnabled": {
+                    "type": "function",
+                    "value": "function javaEnabled() { [native code] }"
+                },
+                "keyboard": {
+                    "type": "object",
+                    "properties": {
+                        "getLayoutMap": {
+                            "type": "function",
+                            "value": "function () { [native code] }"
+                        },
+                        "lock": {
+                            "type": "function",
+                            "value": "function () { [native code] }"
+                        },
+                        "unlock": {
+                            "type": "function",
+                            "value": "function () { [native code] }"
+                        }
+                    }
+                },
+                "language": {
+                    "type": "string"
```

Comment:
why is there no value here?

Reply:
for raw values we don't store the value as the purpose of the profile is just to describe the metadata around the properties.